### PR TITLE
First pass through reviewing the CHANGES file

### PR DESCRIPTION
**Pull request by ronawho**

### CODE REVIEW

**Comment by ronawho** [comment]
@bradcray making my (hopefully) last pass through now, but here are my current mods in the meantime.

**Comment by ronawho** [comment]
Pushed a few other changes, think I'm done now

**Comment by ben-albrecht** [comment]
I can't comment on whether or not these modifications include all changes necessary, but the edits look good.

File: CHANGES [hunk 1]
```diff
@@ -60,7 +60,8 @@ Tool Changes
   - automatically skip documentation of 'private' symbols
   - squashed generation of 'inline' in output
   - added support for --print-commands with chpldoc
-* added a quit() primitive to chpl-ipe for clean exits from the environment
+* chpl-ipe:
+  - added a quit() primitive for clean exits from the environment
```
Comment (bradcray) [inline CHANGES]: Ha!  I had it this way in an earlier draft, but decided that a 1-bullet list wasn't worthwhile, esp. given the lack of work on chpl-ipe this time around.  :)  Not worth changing back for my sake, though.

**Comment by bradcray** [comment]
This looks good to me -- thanks for taking the pass @ronawho.

**Pull request merged**